template/content/server.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env node
+'use strict';
+
+var fileStart = Date.now();
+
+var hostname = require('os').hostname;
+var process = require('process');
+var path = require('path');
+var getRepoInfo = require('git-repo-info');
+var parseArgs = require('minimist');
+
+var Application = require('./app.js');
+
+module.exports = main;
+
+/*istanbul ignore else: dead branch*/
+if (require.main === module) {
+    main({
+        argv: parseArgs(process.argv.slice(2))
+    });
+}
+
+function main(opts) {
+    var mainStart = Date.now();
+
+    /*eslint no-process-exit: 0*/
+    var app = Application();
+
+    // attach before throwing exception
+    process.on('uncaughtException', app.clients.uncaught);
+
+    var gitRepo = path.join(__dirname, '.git');
+    var gitSha = getRepoInfo(gitRepo).sha;
+
+    app.bootstrap(onAppReady);
+
+    var bootstrapEnd = Date.now();
+
+    function onAppReady(err) {
```

Comment:
Poorly named. This is onClientsReady or onDepsReady. Consider moving the closure outside, and passing `(err, opts)` to it with a smaller closure.